firebase-firestore/src/test/java/com/google/firebase/firestore/local/LruGarbageCollectorTestCase.java

```diff
@@ -401,6 +401,30 @@ public void testRemoveOrphanedDocuments() {
         });
   }
 
+  @Test
+  public void testRemoveOrphanedDocumentsExistsEarlyWithNoDocuments() {
```

Comment:
Exists -> Exits ?

Reply:
Also it's not testing `EarlyExits` but the case where no documents are collectable? If my understanding is correct, maybe rename the test?

Reply:
Fixed both issues as once by renaming this to`testRemoveOrphanedDocumentsWithNoDocuments()`